spec/features/guider_views_appointments_spec.rb

```diff
@@ -42,6 +42,78 @@
     end
   end
 
+  context 'Appointment is updated by someone else', js: true do
+    scenario 'viewing their appointments' do
```

Comment:
Can we make the combination of the context and scenario text follow on in proper English? Though ideally we'd just flatten the context into the two scenarios instead I guess. Don't worry about duplication it's incidental.